docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.9"
+services:
+  mongo:
+    image: mongo
+    ports:
+      - 27018:27018
+  web:
+    build: ./backend
```

Comment:
As I mentioned above, use environment variables. You need to define them in `docker-compose.yml` and these variables will be propagated into container and then you can access them from python code (not only actually)